keyboards/redscarf_i/redscarf_i.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2019 Ben Weakley
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_4x6_ortho( \
```

Comment:
This layout should be called `LAYOUT_ortho_6x4` to support the Community layouts feature.
It also would be a good idea to add the numpad layouts here (`numpad_6x4` and `numpad_5x4`).

```suggestion
#define LAYOUT_ortho_6x4( \
```